templates/user/settings/keys_ssh.tmpl

```diff
@@ -35,6 +31,11 @@
 		<div class="item">
 			{{.locale.Tr "settings.ssh_desc"}}
 		</div>
+		{{if .DisableSSH}}
+			<div class="item">
```

Comment:
Likely should be a warning message, e.g. https://fomantic-ui.com/collections/message.html#warning

Please show a screenshot.

Reply:
Unlikely to require a warning level, however I'm not sure. 

I can send a screenshot tomorrow

Reply:
Yeah, I haven't tested yet. Depends on UI context if a warning box is warranted.

Reply:
![grafik](https://user-images.githubusercontent.com/19504461/230869099-5f60f74e-000a-428d-b5dc-8699faaa2efb.png)

Reply:
Okay, seems good without any "severity" there.